### PR TITLE
Added Italian translation for PluginManager addon

### PR DESCRIPTION
**Pull request by giansalvo**
Add file it-local.po

### CODE REVIEW

File: PluginManager/po/it-local.po [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Last-Translator: Giansalvo Gusinu\n"
+"Language-Team: \n"
+"X-Generator: Poedit 2.2.1\n"
+
+#: PluginManager/PluginManager.gpr.py:29
+msgid "Plugin Manager Enhanced"
+msgstr "Gestore Plugin Avanzato"
```
Comment (tosky) [inline PluginManager/po/it-local.po]: lowercase: Gestore plugin avanzato

File: PluginManager/po/it-local.po [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+
+#: PluginManager/PluginManager.gpr.py:30
+msgid "An Addon/Plugin Manager with several additional capabilities"
+msgstr "Un Gestore di Addon/Plugin con varie funzionalità addizionali"
```
Comment (tosky) [inline PluginManager/po/it-local.po]: "Un gestore di componenti aggiuntivi/plugin con varie funzionalità addizionali"
(addon -> componente aggiuntivi; plugin -> plugin)

File: PluginManager/po/it-local.po [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+
+#: PluginManager/PluginManager.py:87
+msgid "Plugin Manager - Enhanced"
+msgstr "Gestore dei Plugin - Avanzato"
```
Comment (tosky) [inline PluginManager/po/it-local.po]: Gestore dei plugin - Avanzato
(but I would go with lowercase as well for avanzato)

File: PluginManager/po/it-local.po [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+msgstr "_Aiuto"
+
+#: PluginManager/PluginManager.py:151
+msgid ""
+"Enter search words to filter the addons.\n"
+"All the words must be present somewhere in the row or\n"
+"the addon filename to be included in the search.\n"
+"Word case and order is ignored."
+msgstr ""
+"Inserisce le parole da cercare per filtrare gli addons.\n"
```
Comment (tosky) [inline PluginManager/po/it-local.po]: addon -> componente aggiuntivo

File: PluginManager/po/it-local.po [hunk 5]
```diff
@@ -0,0 +1,265 @@
...
+
+#: PluginManager/PluginManager.py:207
+msgid "Restart..."
+msgstr "Riavvia..."
+
+#: PluginManager/PluginManager.py:208
+msgid ""
+"Please Restart Gramps so that your addon changes can be safely completed."
+msgstr ""
+"Si prega di Riavviare Gramps in modo da completare l'aggiornamento degli "
```
Comment (tosky) [inline PluginManager/po/it-local.po]: Without "si prega", the "please" form is English only:
Riavviare Gramps in modo da completare l'aggionramento dei componenti aggiuntivi in maniera sicura.

File: PluginManager/po/it-local.po [hunk 6]
```diff
@@ -0,0 +1,265 @@
...
+"il nome dell'addon da includere nella ricerca.\n"
+"Maiuscole/minuscole e l'ordine delle parole sono ignorate."
+
+#: PluginManager/PluginManager.py:155
+msgid "Search..."
+msgstr "Ricerca..."
+
+#: PluginManager/PluginManager.py:161
+msgid "Check for updated addons now"
+msgstr "Cerca gli addons aggiornati adesso"
```
Comment (tosky) [inline PluginManager/po/it-local.po]: see above, and I would make it more linear:
"Cerca adesso i componenti aggiuntivi aggiornati"

File: PluginManager/po/it-local.po [hunk 7]
```diff
@@ -0,0 +1,265 @@
...
+#: PluginManager/PluginManager.py:208
+msgid ""
+"Please Restart Gramps so that your addon changes can be safely completed."
+msgstr ""
+"Si prega di Riavviare Gramps in modo da completare l'aggiornamento degli "
+"addons in maniera sicura."
+
+#: PluginManager/PluginManager.py:284
+msgid "Id"
+msgstr ""
```
Comment (tosky) [inline PluginManager/po/it-local.po]: Id can stay as it is

File: PluginManager/po/it-local.po [hunk 8]
```diff
@@ -0,0 +1,265 @@
...
+msgid "Version"
+msgstr "Versione"
+
+#: PluginManager/PluginManager.py:288
+msgid "Authors"
+msgstr "Autori"
+
+#: PluginManager/PluginManager.py:289
+msgid "Email"
+msgstr ""
```
Comment (tosky) [inline PluginManager/po/it-local.po]: Can be translated as it is

File: PluginManager/po/it-local.po [hunk 9]
```diff
@@ -0,0 +1,265 @@
...
+
+#: PluginManager/PluginManager.py:290
+msgid "Filename"
+msgstr "Nome del file"
+
+#: PluginManager/PluginManager.py:291
+msgid "Location"
+msgstr "Posto"
```
Comment (tosky) [inline PluginManager/po/it-local.po]: Località

File: PluginManager/po/it-local.po [hunk 10]
```diff
@@ -0,0 +1,265 @@
...
+msgid "Loaded"
+msgstr "Caricato"
+
+#: PluginManager/PluginManager.py:299
+msgid "Hidden"
+msgstr "Nascosto"
+
+#: PluginManager/PluginManager.py:304 PluginManager/PluginManager.py:699
+msgid "Failed"
+msgstr "Fallito"
```
Comment (tosky) [inline PluginManager/po/it-local.po]: Non riuscito

File: PluginManager/po/it-local.po [hunk 11]
```diff
@@ -0,0 +1,265 @@
...
+
+#: PluginManager/PluginManager.py:377
+msgid "Installation Errors"
+msgstr "Errori di installazione"
+
+#: PluginManager/PluginManager.py:378
+msgid "The following addons had errors: "
+msgstr "Gli addons seguente presentano errori: "
```
Comment (tosky) [inline PluginManager/po/it-local.po]: see above

File: PluginManager/po/it-local.po [hunk 12]
```diff
@@ -0,0 +1,265 @@
...
+#: PluginManager/PluginManager.py:438
+msgid "Status"
+msgstr "Stato"
+
+#: PluginManager/PluginManager.py:441
+msgid ""
+"'*' items are supplied by 3rd party authors,\n"
+"<s>strikeout</s> items are hidden"
+msgstr ""
+"'*' elementi fornite da autori di terze parti,\n"
```
Comment (tosky) [inline PluginManager/po/it-local.po]: "Gli elementi '*' sono forniti da autori di terze parti,\n"
"gli <s>elementi barrati</s> sono nascosti"

File: PluginManager/po/it-local.po [hunk 13]
```diff
@@ -0,0 +1,265 @@
...
+msgstr "Mostra gli elementi nascosti"
+
+#: PluginManager/PluginManager.py:502
+#, fuzzy
+msgid "Show Built-in items"
+msgstr "Mostra gli elementi interni"
+
+#: PluginManager/PluginManager.py:509
+msgid "* indicates 3rd party addon"
+msgstr "* indica gli addons di terze parti"
```
Comment (tosky) [inline PluginManager/po/it-local.po]: see above

File: PluginManager/po/it-local.po [hunk 14]
```diff
@@ -0,0 +1,265 @@
...
+
+#: PluginManager/PluginManager.py:514
+msgid "Plugins"
+msgstr ""
```
Comment (tosky) [inline PluginManager/po/it-local.po]: Plugin (without s, which is not used in Italian for loaned words)

File: PluginManager/po/it-local.po [hunk 15]
```diff
@@ -0,0 +1,265 @@
...
+msgstr "Mostra"
+
+#: PluginManager/PluginManager.py:603
+msgid "Uninstall"
+msgstr "Disinstalla"
+
+#: PluginManager/PluginManager.py:613
+#, fuzzy
+msgid "Checking Addons Failed"
+msgstr "Il controllo degli addons è fallito"
```
Comment (tosky) [inline PluginManager/po/it-local.po]: see above

File: PluginManager/po/it-local.po [hunk 16]
```diff
@@ -0,0 +1,265 @@
...
+
+#: PluginManager/PluginManager.py:614
+#, fuzzy
+msgid "The addon repository appears to be unavailable. Please try again later."
+msgstr ""
+"Il database degli addons non è disponibile. Si prega di riprovare più tardi."
```
Comment (tosky) [inline PluginManager/po/it-local.po]: see above (also, I wouldn't use "database" for "repository", the meaning of the latter is more specific)

File: PluginManager/po/it-local.po [hunk 17]
```diff
@@ -0,0 +1,265 @@
...
+
+#: PluginManager/PluginManager.py:628
+msgid ""
+"3rd party addons are not shown until the addons update list is downloaded.  "
+"Would you like to check for updated addons now?"
+msgstr ""
+"Gli addons di terze parti non sono mostrati fino allo scaricamento della "
```
Comment (tosky) [inline PluginManager/po/it-local.po]: see above (and elenco for lista)

File: PluginManager/po/it-local.po [hunk 18]
```diff
@@ -0,0 +1,265 @@
...
+"lista aggiornata degli addons. Si desidera verificare la presenza di addons "
+"aggiornati adesso?"
+
+#: PluginManager/PluginManager.py:631
+msgid "Yes"
+msgstr "Si"
+
+#: PluginManager/PluginManager.py:631
+msgid "No"
+msgstr ""
```
Comment (tosky) [inline PluginManager/po/it-local.po]: please translate it, even if it is the same value

File: PluginManager/po/it-local.po [hunk 19]
```diff
@@ -0,0 +1,265 @@
...
+"3rd party addons are not shown until the addons update list is downloaded.  "
+"Would you like to check for updated addons now?"
+msgstr ""
+"Gli addons di terze parti non sono mostrati fino allo scaricamento della "
+"lista aggiornata degli addons. Si desidera verificare la presenza di addons "
+"aggiornati adesso?"
+
+#: PluginManager/PluginManager.py:631
+msgid "Yes"
+msgstr "Si"
```
Comment (tosky) [inline PluginManager/po/it-local.po]: Sì (note the accent on the ì)

File: PluginManager/po/it-local.po [hunk 20]
```diff
@@ -0,0 +1,265 @@
...
+msgstr "Carica"
+
+#: PluginManager/PluginManager.py:494
+msgid "Show hidden items"
+msgstr "Mostra gli elementi nascosti"
+
+#: PluginManager/PluginManager.py:502
+#, fuzzy
+msgid "Show Built-in items"
+msgstr "Mostra gli elementi interni"
```
Comment (tosky) [inline PluginManager/po/it-local.po]: Not sure whether to use interno or integrato

File: PluginManager/po/it-local.po [hunk 21]
```diff
@@ -0,0 +1,265 @@
...
+msgstr "*Installato"
+
+#: PluginManager/PluginManager.py:702
+msgid "Update Available"
+msgstr "Aggiornamento disponibile"
+
+#: PluginManager/PluginManager.py:729
+#, python-format
+msgid "%(str1)s: %(str2)s"
+msgstr ""
```
Comment (tosky) [inline PluginManager/po/it-local.po]: please translate it even if it is the same

File: PluginManager/po/it-local.po [hunk 22]
```diff
@@ -0,0 +1,265 @@
...
+msgid "Detailed Info"
+msgstr "Informazioni dettagliate"
+
+#: PluginManager/PluginManager.py:828
+msgid "Updated"
+msgstr "Aggiornato"
+
+#: PluginManager/PluginManager.py:839
+msgid "updates|New"
+msgstr "aggiornamenti|Nuovo"
```
Comment (tosky) [inline PluginManager/po/it-local.po]: Nuovo
(the string before | is the context and it must not be translated)

**Comment by giansalvo** [comment]
Pushed new file to take into account the changes requested

File: PluginManager/po/it-local.po [hunk 23]
```diff
@@ -182,17 +183,16 @@ msgid "Show hidden items"
...
 
 #: PluginManager/PluginManager.py:502
-#, fuzzy
 msgid "Show Built-in items"
 msgstr "Mostra gli elementi interni"
 
 #: PluginManager/PluginManager.py:509
 msgid "* indicates 3rd party addon"
-msgstr "* indica gli addons di terze parti"
+msgstr "Gli elementi '*' sono forniti da autori di terze parti"
```
Comment (tosky) [inline PluginManager/po/it-local.po]: The change requested here for about addons, not the *:
'* indica i componenti aggiuntivi di terze parti"

File: PluginManager/po/it-local.po [hunk 24]
```diff
@@ -21,15 +21,16 @@ msgstr ""
...
 #: PluginManager/PluginManager.gpr.py:29
 msgid "Plugin Manager Enhanced"
-msgstr "Gestore Plugin Avanzato"
+msgstr "Gestore plugin avanzato"
 
 #: PluginManager/PluginManager.gpr.py:30
 msgid "An Addon/Plugin Manager with several additional capabilities"
-msgstr "Un Gestore di Addon/Plugin con varie funzionalità addizionali"
+msgstr ""
+"Un Gestore di componenti aggiuntivi/plugin con varie funzionalità addizionali"
```
Comment (tosky) [inline PluginManager/po/it-local.po]: Un gestore di...
(Gestore lowercase)

**Comment by sam-m888** [comment]
@giansalvo Do you have time to finish the requested changes from @tosky ?

**Comment by tosky** [comment]
Uhm, I guess I can commit it as it is and push a follow-up fix.

**Comment by giansalvo** [comment]
Yes, I'll do it this evening at last.

**Comment by tosky** [comment]
Thank you!
@sam-m888 , @prculley, should I backport the changes to maintainance/gramps50, or is it something that you do regularly?

**Comment by sam-m888** [comment]
> should I backport the changes to maintainance/gramps50

@tosky If you could please. Then I'll rebuild the addon for both branches. Thank you

**Comment by tosky** [comment]
Backported in d234469

**Comment by sam-m888** [comment]
> Backported in d234469

@tosky Both rebuilt. Thank you 👍

**Pull request merged**